pom.xml

```diff
@@ -19,9 +19,9 @@
   <properties>
     <revision>5.22</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jetty.version>9.4.44.v20210927</jetty.version>
+    <jetty.version>10.0.7</jetty.version>
     <alpn.api.version>1.1.3.v20160715</alpn.api.version>
-    <java.level>8</java.level>
+    <java.target>11</java.target>
```

Comment:
This change has no effect (see for yourself by setting `<java.target>nonsense</java.target>`. I suggest reverting.

Reply:
yup the parent pom and format of this is a real pain... I have started some changed in a local parent pom (but I havent commit it)